core/pom.xml

```diff
@@ -25,6 +25,11 @@
     </plugins>
   </build>
   <dependencies>
+    <dependency>
+      <groupId>io.jenkins.fields</groupId>
+      <artifactId>fields</artifactId>
+      <version>1.0.0-20180620.094913-3</version>
```

Comment:
Please link to upstream PR, or otherwise give some indication in the POM of how we would know when this is ready to bump to a release version.